packages/unlock-js/src/web3Service.ts

```diff
@@ -699,4 +699,41 @@ export default class Web3Service extends UnlockService {
     })
     return price
   }
+
+  /**
+   * Returns freeTrialLength value
+   */
+  async freeTrialLength({
+    lockAddress,
+    network,
+  }: {
+    lockAddress: string
+    network: number
+  }) {
+    const lockContract = await this.getLockContract(
+      lockAddress,
+      this.providerForNetwork(network)
+    )
+    const freeTrialLength = await lockContract.freeTrialLength()
```

Comment:
Generally I would rather you version(ed) implementations. but I guess it is ok if it is the same everywhere.

Reply:
this is defined in the same way for all the version in `abis`